docs/source/getting_started.rst

```diff
@@ -0,0 +1,15 @@
+Getting started
+===============
+
+New model can be generated using the command-line tool from the
+terminal:
+
+::
+
+    ivadomed ivadomed/config/config.json
+
+where ``config.json`` is a configuration file, which parameters are
+described in the :ref:`configuration_file:Configuration File`.
+
+To fully benefit from all the features of ``ivadomed``, please see the
+:ref:`tutorials:Tutorials`.
```

Comment:
admonition to fix: https://ivadomed.org/en/aj-286-convert-docs-to-rst/tutorials.html

Reply:
Locally I see this:

![image](https://user-images.githubusercontent.com/7751484/84913090-20c45500-b088-11ea-8e3c-5fa2a2163851.png)

Probably a discrepancy with the build tools..? This is going to be fun to debug..

Reply:
Changed it to a `note` admonition to see and readthedocs renders it correctly. Still puzzled as to why the `warning` admonition doesn't render properly...

Reply:
find with note-- in fact, this is better